## Code Review

### Critical

- `components/ui/calendar.tsx:89` — CI: **TypeScript Check** is still failing after this PR. The `rootRef` removal was correct, but `table` is a second class name key that was removed from v10's `ClassNames` type — `error TS2353: Object literal may only specify known properties, and 'table' does not exist in type 'Partial<ClassNames>'`. The fix must also remove (or rename) the `table` key in the `classNames` prop passed to `<DayPicker>`. In v10 the equivalent key is `month_grid`.

- CI: **Vercel** check is failing — check name: `Vercel`, link: https://vercel.com/sga-operational-affairs-projects/aplio/4Hre8mtLeP3gifF7wGtkGV9mADDu. This is likely a downstream consequence of the TSC failure; the build cannot succeed while the type error exists.

### Low

- `components/ui/calendar.tsx` — The plan's implementation checklist item "Run `npm run tsc:check` locally in the worktree to confirm the error is resolved" was not completed before pushing — the remaining type error (`table`) would have surfaced immediately with a local check. Completing pre-push checks before opening the PR (`npm run tsc:check`, `npm run prettier:check`, `npm run eslint:check`) is required per `CLAUDE.md`.

---
_Reviewed against plan in issue #115_

--- a/components/ui/calendar.tsx
+++ b/components/ui/calendar.tsx
@@ -130,14 +130,9 @@ function Calendar({
         ...classNames,
       }}
       components={{
-        Root: ({ className, rootRef, ...props }) => {
+        Root: ({ className, ...props }) => {
           return (
-            <div
-              data-slot="calendar"
-              ref={rootRef}
-              className={cn(className)}
-              {...props}
-            />
+            <div data-slot="calendar" className={cn(className)} {...props} />
           );
         },
         Chevron: ({ className, orientation, ...props }) => {

--- a/components/ui/calendar.tsx
+++ b/components/ui/calendar.tsx
@@ -86,7 +86,7 @@ function Calendar({
             : 'rounded-md pl-2 pr-1 flex items-center gap-1 text-sm h-8 [&>svg]:text-muted-foreground [&>svg]:size-3.5',
           defaultClassNames.caption_label,
         ),
-        table: 'w-full border-collapse',
+        month_grid: 'w-full border-collapse',
         weekdays: cn('flex', defaultClassNames.weekdays),
         weekday: cn(
           'text-muted-foreground rounded-md flex-1 font-normal text-[0.8rem] select-none',

--- a/package.json
+++ b/package.json
@@ -41,7 +41,7 @@
     "clsx": "^2.1.1",
     "cmdk": "^1.1.1",
     "lucide-react": "^1.17.0",
-    "react-day-picker": "^9.14.0",
+    "react-day-picker": "^10.0.1",
     "next": "16.2.9",
     "next-themes": "^0.4.6",
     "nuqs": "^2.8.9",

